app/src/components/buttons/ghostButton/ghostButton.scss

```diff
@@ -64,6 +64,42 @@
       color: $COLOR--white-two;
     }
   }
+
+  &.color-gray-91:not(.disabled) {
```

Comment:
Looks like a copypaste. The only difference is the border color. Can it be achieved by adding a new prop `grayBorder` or something like that?